changelog/_unreleased/2025-11-06-fix-app-lifecycle-with-custom-entity-associations.md

```diff
@@ -0,0 +1,11 @@
+---
+title: fix app lifecycle with custom entity associations
+issue: 13273
+---
+# Core
+* Added a new optional attribute, `ignore-missing-reference` for custom entity field associations in the schema definition. This allows the schema updater to skip creating associations if the referenced table does not exist, improving flexibility and robustness during schema updates.
+
```

Comment:
Could you add information here, what where the class touched? follow _example.md